prac_09/file_to_sort_v1.py

```diff
@@ -0,0 +1,20 @@
+import os
+import shutil
+
+
+def main():
+    # Move to desired directory
+    os.chdir('FilesToSort')
+    print("Current directory is", os.getcwd())
+
+    for dir_name, subdir_list, file_list in os.walk('.'):
+        for f in file_list:
```

Comment:
`f` is commonly used as a file _object_, but here it is a file _name_. The variable name should be clearer.